Conceptpower+Spring/src/test/java/edu/asu/conceptpower/core/ConceptManagerTest.java

```diff
@@ -135,7 +135,7 @@ public void testAddConceptListEntry()
         newConcept.setWord("test new");
         newConcept.setWordnetId("WID-1");
         String id = managerToTest.addConceptListEntry(newConcept);
-        Mockito.verify(indexService).insertConcept(newConcept);
+        Mockito.verify(indexService, Mockito.times(2)).insertConcept(newConcept);
```

Comment:
Is there a reason that this is called twice? I think that was a mixup while merging.